src/members/templates/members/set/set_password.html

```diff
@@ -18,6 +24,8 @@ <h1>Set Password Page</h1>
         {% endfor %}
     </ul>
     {% endif %}
+    <h5>Do you already have an account?</h5>
+    <button type="button"><a href="{% url 'login' %}">Login Existing Account</a></button>
```

Comment:
Nice one! Can you add something similar to the login page to redirect the user to the register page to create an account?